clang/include/clang/Basic/BuiltinsDPU.def

```diff
@@ -15,5 +15,7 @@
 // The format of this database matches clang/Basic/Builtins.def.
 
 BUILTIN(__builtin_dpu_tid, "UiC", "nc")
+BUILTIN(__builtin_sdma_rri, "vv*v*255Ui", "")
+BUILTIN(__builtin_ldma_rri, "vv*v*255Ui", "")
```

Comment:
Do we want to use other names than the signature for the builtins?
Also, do we want to add a "dpu" prefix, like `__builtin_dpu_tid`?

Reply:
We should not use ``__builtin_sdma_rri``.
I agree, we want to have ``dpu`` in the name.
And as this builtin does not match directly this dpu instruction (``sdma_rri``), we should not call it that way.
(It does not match it in the case where the size is known dynamically and we need to add instruction to insert the transfer size in the first argument of the instruction)

Maybe ``__builtin_dpu_sdma`` ?

Reply:
I am ok with that.
I do not think we would gain much clarity with something like `__builtin_dpu_wram2mram_xfr`.

Reply:
Done